specification/compute/resource-manager/Microsoft.Compute/stable/2021-11-01/compute.json

```diff
@@ -8941,6 +8941,12 @@
           "$ref": "#/definitions/DedicatedHostInstanceView",
           "readOnly": true,
           "description": "The dedicated host instance view."
+        },
+        "timeCreated": {
+          "readOnly": true,
+          "type": "string",
+          "format": "date-time",
+          "description": "Specifies the time at which the Dedicated Host resource was created."
```

Comment:
Add the min version from when this property will be available.

Reply:
Added